examples/mip/sudoku.py

```diff
@@ -0,0 +1,65 @@
+# Inspired by https://www.gurobi.com/documentation/9.5/examples/sudoku_py.html
+
+import math
+
+import numpy as np
+
+from symfit import Parameter, symbols, IndexedBase, Idx, Sum, Eq
+from symfit.symmip.backend import GurobiBackend
+from symfit import MIP
+
+with open('sudoku1') as f:
+    grid = f.read().split()
+
+for line in grid:
+    print(line)
+
+n = len(grid[0])
+s = math.isqrt(n)
+
+# Fix variables associated with cells whose values are pre-specified
+# lb = np.array([[0 if char == "." else 1 for j, char in enumerate(line)] for i, line in enumerate(grid)])
+lb = np.zeros((n, n, n), dtype=int)
+for i in range(n):
+    for j in range(n):
+        if grid[i][j] != '.':
+            v = int(grid[i][j]) - 1
+            lb[i, j, v] = 1
+ub = np.ones_like(lb)
```

Comment:
I don't like these variable names.
Some explanation as to why the one-hot encoding also wouldn't be misplaced in an example